include/mbedtls/ecp.h

```diff
@@ -419,11 +419,22 @@ typedef struct mbedtls_ecp_keypair {
 }
 mbedtls_ecp_keypair;
 
-/*
- * Point formats, from RFC 4492's enum ECPointFormat
+/**
+ * The uncompressed point format for Short Weierstrass curves
+ * (MBEDTLS_ECP_DP_SECP* and MBEDTLS_ECP_DP_BP*). //no-check-names
```

Comment:
This includes “//no-check-names” in the documentation. That's not good.

Our normal convention is to write `XXX` or `xxx` for a wildcard, not `*`. And `check_names.py` has an exception for names containing these.

```suggestion
 * (MBEDTLS_ECP_DP_SECP_XXX and MBEDTLS_ECP_DP_BP_XXX).
```
and likewise below.